src/verify_cert.rs

```diff
@@ -399,6 +405,39 @@ fn check_eku(
     }
 }
 
+type KeyUsageBit = usize;
```

Comment:
I tend to dislike type aliases for this type of thing: they obfuscate but don't provide type safety. In this case, I think a translation to an enum would be more idiomatic?

```rust
#[repr(u8)]
enum KeyUsageMode {
    // DigitalSignature = 0,
    // ContentCommitment = 1,
    CrlSign = 6,
    ..
}
```

Reply:
Sounds good, I think using `#[repr(u8)]` was the missing piece that kept me from thinking of this initially.

Reply:
Updated to remove the type alias in place of an enum. Thanks!